packages/blade/src/components/Radio/__tests__/Radio.web.test.tsx

```diff
@@ -64,4 +65,39 @@ describe('<Radio />', () => {
     await user.click(getByLabelText(fruitApple));
     expect(getByRole('radio', { name: fruitApple })).toBeChecked();
   });
+
+  it(`should expose native element methods via ref`, async () => {
+    const label = 'Accept';
+
+    const Example = (): React.ReactElement => {
+      const ref = React.useRef<HTMLInputElement>(null);
+
+      return (
+        <>
+          <RadioGroup label="focus test">
```

Comment:
how does focus work in the case of the Radio and Checkbox group? does the 1st child gets the focus?

Reply:
RadioGroup is not focusable. Radio only has the input. 

Users will add the ref to the Radio component and handle the focus if they need to. 

In this test I've used RadioGroup just because Radio cannot be used outside of RadioGroup